librz/core/linux_heap_jemalloc.c

```diff
@@ -176,7 +176,7 @@ static void GH(jemalloc_get_chunks)(RzCore *core, const char *input) {
 		extent_node_t *head = RZ_NEW0(extent_node_t);
 
 		if (!node || !head) {
-			RZ_LOG_ERROR("Error calling calloc\n");
+			RZ_LOG_ERROR("Cannot call calloc\n");
```

Comment:
this error is misleading. it does call `calloc` (via `RZ_NEW0`) but doesn't allocate the type `extent_node_t`
```suggestion
			RZ_LOG_ERROR("Failed to allocate extent_node_t\n");
```